src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -421,7 +421,7 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                         const parentDocument = domUtils.findDocument(parentEl);
                         const parentWindow   = parentDocument ? parentDocument.defaultView : null;
 
-                        el.outerHTML = processHtml('' + value, parentEl.tagName);
+                        el.outerHTML = processHtml({ html: '' + value, parentTag: parentEl.tagName });
```

Comment:
Use `String` constructor instead of `'' + value`.
Here and below.

`processHtml({ html: String(value), parentTag: parentEl.tagName })`